### PR TITLE
Add ImagePullBackOff + update format

### PR DESCRIPTION
**Pull request by abeatrix**
Add troubleshooting steps for ImagePullBackOff & 429 Too Many Requests error to K8s troubleshooting docs based on @ggilmore's reply in customer issue #328

### CODE REVIEW

File: doc/admin/install/kubernetes/troubleshoot.md [hunk 1]
```diff
@@ -109,8 +109,13 @@ If you're using Google Cloud Platform, note that the default node type is `n1-st
 with only one CPU, and that some components request a 2-CPU node. When creating a cluster, use
 `--machine-type=n1-standard-16`.
 
-> You can't access Sourcegraph.
+#### `ImagePullBackOff` / `429 Too Many Requests`
+This indicates the instance is getting rate-limited by Docker Hub([link](https://www.docker.com/increase-rate-limits)), where our images are stored, as unauthenticated users are limited to 100 image pulls within a 6 hour period. Possible solutions included:
+1. Wait until the rate limits are reset
+2. Upgrade your account to a Docker Pro or Team subscription ([See Docker Hub for more information](https://www.docker.com/increase-rate-limits))
```
Comment (daxmc99) [inline doc/admin/install/kubernetes/troubleshoot.md:115]: I would put this last and mention that it is optional. 
Setting ImagePullSecrets should get you most of the way

File: doc/admin/install/kubernetes/troubleshoot.md [hunk 2]
```diff
@@ -109,8 +109,13 @@ If you're using Google Cloud Platform, note that the default node type is `n1-st
...
+3. Create a Docker Hub account with a higher rate limit > configure an `ImagePullSecrets` K8S object with your Docker Hub service that contains your docker credentials ([link to tutorial](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)) > add these credentials to the default service account that's running the same namespace that Sourcegraph is running in ([link to tutorial](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account))
```
Comment (daxmc99) [inline doc/admin/install/kubernetes/troubleshoot.md:116]: nit: change 3 to 1, markdown automatically increments numbered lists

**Review by daxmc99 — CHANGES_REQUESTED** [review]

File: doc/admin/install/kubernetes/troubleshoot.md (hunk 2 above)
Comment (abeatrix) [reply doc/admin/install/kubernetes/troubleshoot.md]: rearranged as suggested.

**Review by abeatrix — COMMENTED** [review]
Rearrange items

**Review by daxmc99 — APPROVED** [review]
LGTM

**Pull request merged**